tests/other_api/controllers/cpp_import_device/cpp_import_device.cpp

```diff
@@ -0,0 +1,32 @@
+#include <webots/DistanceSensor.hpp>
+#include <webots/Field.hpp>
+#include <webots/Robot.hpp>
+#include <webots/Supervisor.hpp>
+
+#include "../../../lib/TestUtils.hpp"
+
+using namespace webots;
+
+int main(int argc, char **argv) {
+  Supervisor *robot = new Supervisor();
+  TestUtils ts(robot, argv[0]);
+
+  int timeStep = (int)robot->getBasicTimeStep();
+  Field *childrenField = robot->getSelf()->getField("children");
+
+  // Import the first PositionSensor
+  childrenField->importMFNodeFromString(0, "DistanceSensor { name \"ds1\" }");
+  DistanceSensor *ds1 = robot->getDistanceSensor("ds1");
+  ts.assertPointerNotNull(ds1, "DistanceSensor 2 not found after import.");
+  ds1->enable(timeStep);
+
+  robot->step(timeStep);
```

Comment:
Is this step needed here? If not, we should remove it. Otherwise we should explain why it is needed.

Reply:
It is not needed. I added just because it is much easier to test if we apply one action per step.
But it can be removed without any problem.

Reply:
I believe it's better to remove it, as it seems useless, e.g., a user would probably not call step in there. To me, it looks like a hack to work around a bug 😉.

Reply:
I removed it.
But the bug was not affected in any way by the step.
It showed independently from the call of the step function.